.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: publish docker image
+on: 
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build/Push image ghcr.io/mcorbin/cabourotte:latest
```

Comment:
imo this part could be removed, I don't plan to support ghcr + docker hub for the image

Reply:
I've removed this part